README.md

```diff
@@ -1,62 +1,86 @@
 # cookiecutter-awesome
-Cookiecutter to create an [awesome][awesome] list.
+Cookiecutter to create an [awesome][awesome-list] list.
 
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
 - [Features](#features)
 - [Requirements](#requirements)
 - [Usage](#usage)
-- [Documentation](#documentation)
+- [Example](#example)
 - [Development](#development)
-- [History](#history)
+  - [History](#history)
+  - [Community](#community)
+- [Other Awesome Lists](#other-awesome-lists)
 - [Credits](#credits)
 - [License](#license)
 
 ## Features
+
 - Creative Commons CC0 1.0 Universal License (as recommended in awesome's
-  [manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md)).
+  [manifesto][manifesto]).
 - Travis configuration with [awesome_bot][awesome_bot] check.
 - Changelog, code of conduct, contributing, and pull request template files.
 - Compliant with awesome's [PR requirements][pr-requirements].
 
 ## Requirements
+
 - [git](https://git-scm.com/downloads)
 - [python](https://www.python.org/downloads/)
 - [cookiecutter](https://github.com/audreyr/cookiecutter)
 - ([awesome_bot][awesome_bot])
 
 ## Usage
+
 ```bash
-cookiecutter gh:Pawamoy/cookiecutter-awesome
+cookiecutter gh:moodule/cookiecutter-awesome
 # OR
-cookiecutter https://github.com/Pawamoy/cookiecutter-awesome
+cookiecutter https://github.com/moodule/cookiecutter-awesome
 ```
 
 Once your project has been generated:
 - Run `awesome_bot README.md` to verify the links.
 - Activate your repository in your Travis account.
 - **Please read [the PR requirements][pr-requirements]**
-  before submitting a Pull Request to [awesome][awesome].
+  before submitting a Pull Request to [awesome][awesome-list].
 
 ## Example
-See the [awesome-repository](https://github.com/Pawamoy/awesome-repository) list.
+
+See the [awesome-repository][awesome-list] list.
```

Comment:
I get that, but all the awesome lists listed in sindresorhus' list were not generated with this cookiecutter, so does it really make sense to link it? I previously linked my own `awesome-repository` list because (self-advertisement heh) I actually generated it with this cookiecutter.

Reply:
I guess it can go either way!

The one from sindresorhus is about "awesome content" and yours is about "contributing to awesome content", so they are complementary i guess.

So I'd put both in here actually! May-be move yours to the contributing part?

I didn't read your list when I first did this, it was a straight steal hahaha

Reply:
Let's put both then!

Reply:
I got lost here !!
indeed only yours should be listed in the root repo, since the awesome-awesome was not made with the cookiecutter...

so yeah I went back to your version :D